File: MPI/mpicheck/tests/functions/4.0/MPI_Buffer_detach_c.f90

        
        program main
        use mpi
        
        TYPE(INTEGER) var_0(10)
       INTEGER(KIND=MPI_COUNT_KIND) var_1
       INTEGER var_2
        call mpi_buffer_detach_c(var_0, var_1, var_2)
       call pmpi_buffer_detach_c(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Buffer_detach_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    void *var_0;
    MPI_Count *var_1;
    int ret;
    /* calls */
    ret = MPI_Buffer_detach_c(var_0, var_1);
    ret = PMPI_Buffer_detach_c(var_0, var_1);
    return 0;
}
